[2024-06-07T06:43:22.459+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_toll_data_16.load_to_database manual__2024-06-07T06:41:42.252474+00:00 [queued]>
[2024-06-07T06:43:22.499+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_toll_data_16.load_to_database manual__2024-06-07T06:41:42.252474+00:00 [queued]>
[2024-06-07T06:43:22.500+0000] {taskinstance.py:1331} INFO - Starting attempt 2 of 3
[2024-06-07T06:43:22.527+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): load_to_database> on 2024-06-07 06:41:42.252474+00:00
[2024-06-07T06:43:22.535+0000] {standard_task_runner.py:57} INFO - Started process 213 to run task
[2024-06-07T06:43:22.544+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_toll_data_16', 'load_to_database', 'manual__2024-06-07T06:41:42.252474+00:00', '--job-id', '342', '--raw', '--subdir', 'DAGS_FOLDER/ETL_toll_data.py', '--cfg-path', '/tmp/tmpzfhvv1yb']
[2024-06-07T06:43:22.550+0000] {standard_task_runner.py:85} INFO - Job 342: Subtask load_to_database
[2024-06-07T06:43:22.715+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_toll_data_16.load_to_database manual__2024-06-07T06:41:42.252474+00:00 [running]> on host f8b9d10836c7
[2024-06-07T06:43:23.288+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='naveedcodes@gmail.com' AIRFLOW_CTX_DAG_OWNER='naveed' AIRFLOW_CTX_DAG_ID='ETL_toll_data_16' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2024-06-07T06:41:42.252474+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-07T06:41:42.252474+00:00'
[2024-06-07T06:43:23.296+0000] {logging_mixin.py:149} INFO - database connected successfully
[2024-06-07T06:43:23.297+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-06-07T06:43:23.313+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=ETL_toll_data_16, task_id=load_to_database, execution_date=20240607T064142, start_date=20240607T064322, end_date=20240607T064323
[2024-06-07T06:43:23.341+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-06-07T06:43:23.372+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
